src/components/GlobalFilter/GlobalFilterMenu.tsx

```diff
@@ -114,43 +117,80 @@ export const GlobalFilterDropdown: React.FunctionComponent<GlobalFilterDropdownP
               }
               position="right"
             >
-              <GroupFilter
-                className="chr-c-menu-global-filter__select"
-                selected={selectedTags}
-                isDisabled={isDisabled}
-                groups={
-                  filter.groups?.map((group) => ({
-                    ...group,
-                    items: group.items?.map((item) => ({
-                      ...item,
-                      onClick: (
-                        e: FormEvent | MouseEventHandler<HTMLInputElement> | undefined,
-                        selected: any,
-                        group: number | undefined,
-                        currItem: boolean | undefined,
-                        groupName: string | undefined,
-                        itemName: string
-                      ) => {
-                        generateGlobalFilterEvent(
-                          (selected?.[groupName as string]?.[itemName as string] as Group)?.isSelected as boolean,
-                          item.value
-                        );
-                        // item.onClick?.(e, selected, group, currItem, groupName, itemName);
-                      },
-                    })),
-                  })) || ([] as unknown as any)
-                }
-                onChange={filter.onChange}
-                placeholder={intl.formatMessage(messages.filterByTags)}
-                isFilterable
-                onFilter={filter.onFilter || (() => undefined)}
-                filterBy={filter.filterBy as string}
-                showMoreTitle={intl.formatMessage(messages.showMore)}
-                onShowMore={() => setIsOpen(true)}
-                showMoreOptions={{
-                  isLoadButton: true,
-                }}
-              />
+              <InputGroup>
```

Comment:
There is a lot of extra "stuff" and the file is getting big and messy. Can you extract the group to a separate file?

Reply:
Moved to a separate component